### PR TITLE
[Rule Tuning] Windows DR Tuning - 2

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

Part of #3186 

## Summary

Tunes the following rules:

* Ingress Transfer via Windows BITS
* Remote File Download via PowerShell
* Remote File Copy via TeamViewer
* Potential Credential Access via DCSync
* Kerberos Traffic from Unusual Process

### CODE REVIEW

File: rules/windows/credential_access_kerberoasting_unusual_process.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 creation_date = "2020/11/02"
 integration = ["endpoint", "windows"]
```
Comment (w0rk3r) [inline rules/windows/credential_access_kerberoasting_unusual_process.toml:3]:
```suggestion
integration = ["endpoint"]
```

File: rules/windows/command_and_control_teamviewer_remote_file_copy.toml [hunk 2]
```diff
@@ -113,7 +113,12 @@ type = "eql"
 
 query = '''
 file where host.os.type == "windows" and event.type == "creation" and process.name : "TeamViewer.exe" and
-  file.extension : ("exe", "dll", "scr", "com", "bat", "ps1", "vbs", "vbe", "js", "wsh", "hta")
+  file.extension : ("exe", "dll", "scr", "com", "bat", "ps1", "vbs", "vbe", "js", "wsh", "hta") and
+  not file.path : (
+        "?:\\Users\\*\\AppData\\Local\\Microsoft\\Windows\\INetCache\\*.js",
+        "?:\\Users\\*\\AppData\\Local\\Temp\\TeamViewer\\update.exe",
+        "?:\\Users\\*\\AppData\\Local\\Temp\\?\\TeamViewer\\update.exe"
```
Comment (Aegrah) [inline rules/windows/command_and_control_teamviewer_remote_file_copy.toml:120]: Just wondering, do we have any hashes / signatures for this?

**Review by Aegrah — APPROVED** [review]
LGTM

File: rules/windows/command_and_control_teamviewer_remote_file_copy.toml (hunk 2 above)
Comment (w0rk3r) [reply rules/windows/command_and_control_teamviewer_remote_file_copy.toml:120]: For the executables?

Comment (w0rk3r) [reply rules/windows/command_and_control_teamviewer_remote_file_copy.toml:120]: https://www.virustotal.com/gui/file/9dd791e623503ad40606b7d44a937b0c71b001f1da0e86c38ef3fa9eea3b384c
https://www.virustotal.com/gui/file/169880f16d53b743a8f11b56273f60909173b5063be564d26f4c6b77fbe9579d
https://www.virustotal.com/gui/file/981b4c6937d50cd5b8ea952d26872020a3684f52017c85073e74d3fcede61efd
https://www.virustotal.com/gui/file/52077813ad446166c6775b70bb863345c0d4e2d5a278bfc2f5948cab2e306d98
https://www.virustotal.com/gui/file/2f2bf41beb44c6885a98adb507aee902470e1d7854bd1c20f5360ff951fc46dd

Comment (Aegrah) [reply rules/windows/command_and_control_teamviewer_remote_file_copy.toml:120]: I meant something in the extend of "Can we add a trusted signing source for those update executables, to make sure they are benign update files?".

Comment (Aegrah) [reply rules/windows/command_and_control_teamviewer_remote_file_copy.toml:120]: Those don't exist for file events. Ignore my comment!

**Review by Samirbous — APPROVED** [review]

**Pull request merged**